src/ui/views/ui_app.jade

```diff
@@ -125,7 +125,7 @@ block content
           err.style['text-align'] = 'center';
           err.style['background'] = '#F44336';
           err.style['padding'] = '25px';
-          err.innerText = '#{i18n('UI-WELCOME_ERROR')}';
+          err.innerText = '#{i18n_singleQuoteVar('UI-WELCOME_ERROR')}';
 
```

Comment:
I am wondering would it be better here to assign to double quotes instead to escape the single quote:
err.innerText = **"**#{i18n('UI-WELCOME_ERROR')}**"**;